x/storage/client/cli/tx.go

```diff
@@ -756,11 +756,13 @@ $ %s tx delete-policy 3
 
 func CmdMirrorBucket() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "mirror-bucket [bucket-id]",
+		Use:   "mirror-bucket [bucket-id] [bucket-name]",
```

Comment:
I think it is a bad parameter setting, for example if I provide `bucket-id`, I dont need to provide `bucket-name`.  I can not set the bucket name only for the current implementation.

Reply:
Yes. I changed it to use flags for both `bucket-id` and `bucket-name`.